addon/mixins/ajax-request.js

```diff
@@ -39,11 +39,24 @@ const {
 } = Ember;
 const JSONAPIContentType = 'application/vnd.api+json';
 
+// Header names are case-insensitive
+// https://www.w3.org/Protocols/rfc2616/rfc2616.html
+function retrieveHeader(headers, name) {
+  var found = null;
+  Object.keys(headers).forEach((key) => {
```

Comment:
Hmm, I see. Maybe we could use the one included in the [`Ember.A()` class](http://emberjs.com/api/classes/Ember.Array.html#method_find)?

It looks like some of the issues on Travis are style-related (we shouldn't be using `var`).  I'm not sure about the other ones, I have the branch pulled locally but need to dig into it to figure out why it's failing now.  I'll be doing that when I get home from work tonight.